### PR TITLE
Send cavalcade logs to CloudWatch

### PR DESCRIPTION
**Pull request by joehoyle**
Depends on #12

This send the logs from each job to CloudWatch logs. This is similar to the cavalcade log file we currently send, but in a much more structured. readable way. Each job run will have an entry, along with it's status, stdout and stderr, ala:

### CODE REVIEW

File: lib/cavalcade-runner-to-cloudwatch/plugin.php [hunk 1]
```diff
@@ -106,3 +129,68 @@ function ( $name, $value ) {
...
+/**
+ * Get a configured CloudWatchClient client.
+ *
+ * @return CloudWatchClient
+ */
+function cloudwatch_client() {
+	return get_aws_sdk()->createCloudWatch([
+		'version'     => '2010-08-01',
+		'http'        => [
+			'synchronous' => false,
```
Comment (nathanielks) [inline lib/cavalcade-runner-to-cloudwatch/plugin.php]: Is this meant to be async?

File: load.php [hunk 2]
```diff
@@ -169,3 +171,27 @@ function load_plugins() {
...
+		'version'  => 'latest',
+	];
+
+	if ( defined( 'AWS_KEY' ) ) {
+		$params['credentials'] = [
+			'key'    => AWS_KEY,
+			'secret' => AWS_SECRET,
+		];
+	}
+	$sdk = new \Aws\Sdk( $params );
```
Comment (nathanielks) [inline load.php]: I haven't seen `static` used in a normal function before. To clarify my understanding from the docs, declaring a variable `static` _inside_ the function means the value of the variable will be preserved, and when the function is run again, used? So in this case, it lets the function cache the value of `$sdk`?

Comment (joehoyle) [reply load.php]: Exactly, it's just an easy way to do a one-time-set cache

**Comment by joehoyle** [comment]
Remove sync flag, let's go!

**Pull request merged**